internal/fs/fs.go

```diff
@@ -244,12 +244,20 @@ func createFileCacheHandler(cfg *ServerConfig) (fileCacheHandler *file.CacheHand
 	// gcsfuse related files.
 	cacheLocation = path.Join(cacheLocation, util.FileCache)
 
+	//Panic in case cacheLocation does not have required permissions
```

Comment:
nit: space between comment and first character of comment ("// P")

Reply:
done